src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/sidebar-nav-bs.html

```diff
@@ -1,5 +1,6 @@
+{#- TODO use unique html id generator since components can be included in multiple places -#}
 <nav class="bd-docs-nav bd-links"
-     aria-label="{{ _('Section Navigation') }}">
-  <p class="bd-links__title" role="heading" aria-level="1">{{ _("Section Navigation") }}</p>
+     aria-labelledby="pst-section-navigation-heading">
+  <p id="pst-section-navigation-heading" class="bd-links__title" role="heading" aria-level="1">{{ _("In this section") }}</p>
```

Comment:
I needed to change the label on the nav because it was noisy, causing screen readers to say "Section Navigation navigation." 

I also thought that "Section Navigation" as a label/header was a bit ambiguous. Section of what? section of the site? section of the page?

My first instinct was to change the heading to "Pages," which means it would be announced by screen readers as "Pages navigation." But then I noticed that the navigation for sections of the page was called "On this page," so I thought maybe it makes sense to call this nav group "In this section," leading to the screen reader announcements "On this page navigation" and "In this section navigation," respectively. 

Taken all together, this means that with this PR the PST site has the following navigation landmarks:

- navigation (no label implies top-level site navigation, I think)
- In this section navigation
- Breadcrumb navigation
- On this page navigation

What do folks think? Should it perhaps be "Pages in this section navigation?"

Reply:
just a note that changing translated strings (strings inside `_(...)`) should not be done lightly (they will need to get re-translated, which is not automatic). I'm not insisting to change it back because it seems like you have good reason to do it here.

"section" here means "section of the site", refers to the top-level menu item (user guide is a section, contributor guide is another section, etc). So I think your choice of "in this section" makes sense here.

Reply:
Indeed. I'm already regretting my decision. It's complicating my PR 😭